contrib/PKGBUILD

```diff
@@ -0,0 +1,35 @@
+# Maintainer: Bruno Pagani (a.k.a. ArchangeGabriel) <archange@archlinux.org>
+# Contributor: Mirco Tischler <mt-ml at gmx dot de>
+
+pkgname=fwupd
+pkgver=LATEST
```

Comment:
Hum the `meson.build` file is one version above the tags. So would need to parse for the first case, using the tag won’t work.